pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -589,6 +589,22 @@ func (nc *Controller) doNodeProcessingPassWorker(ctx context.Context) {
 			return
 		}
 		nodeName := obj.(string)
+		if nc.runTaintManager {
+			node, err := nc.nodeLister.Get(nodeName)
+			if node != nil && err == nil {
+				_, currentReadyCondition := controllerutil.GetNodeCondition(&node.Status, v1.NodeReady)
+				if currentReadyCondition != nil && currentReadyCondition.Status == v1.ConditionTrue {
+					removed, err := nc.markNodeAsReachable(ctx, node)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/0a689af469de0b2095c70d53bfbb920ce7f4f9fb/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L1505-L1519
`nc.zoneNoExecuteTainter` may not be initialized for the node here
```
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime.logPanic({0x4268a40?, 0x7cd8d00})
	/home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:75 +0x99
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0x0?})
	/home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:49 +0x75
panic({0x4268a40, 0x7cd8d00})
	/usr/local/go/src/runtime/panic.go:884 +0x212
k8s.io/kubernetes/pkg/controller/nodelifecycle/scheduler.(*RateLimitedTimedQueue).Remove(...)
	/home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/controller/nodelifecycle/scheduler/rate_limited_queue.go:275
k8s.io/kubernetes/pkg/controller/nodelifecycle.(*Controller).markNodeAsReachable(0xc007ced980, {0x5450bb0, 0xc0008c27c0}, 0xc0013b8000)
```